examples/node-xhr/package.json

```diff
@@ -21,6 +16,7 @@
     "cors": "^2.8.4",
     "formidable": "^1.2.1",
     "npm-run-all": "^4.1.3",
-    "rimraf": "^2.6.2"
+    "rimraf": "^2.6.2",
+    "uppy": "^0.30.4"
   }
```

Comment:
I wonder if we need to add each individual uppy module instead of a bundle thing. So like `@uppy/core`, `@uppy/xhr-upload` etc.